tests/wf/hello-workflow.cwl

```diff
@@ -0,0 +1,9 @@
+cwlVersion: v1.0
```

Comment:
The point is to traverse both branches of https://github.com/common-workflow-language/cwltool/blob/23ad7342896e0a46d7e25ee479f764ba60c61983/cwltool/main.py#L658